x-pack/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/auto_follow_pattern_list.js

```diff
@@ -88,6 +95,78 @@ export const AutoFollowPatternList = injectI18n(
       clearInterval(this.interval);
     }
 
+    renderHeader() {
+      const { isAuthorized } = this.props;
+      return (
+        <Fragment>
+          <EuiFlexGroup justifyContent="spaceBetween" alignItems="flexStart">
+            <EuiFlexItem grow={false}>
+              <EuiText>
+                <p>
+                  <FormattedMessage
```

Comment:
Looks like the wrong content was copied here. It should be this:

![image](https://user-images.githubusercontent.com/1238659/51957072-03dda200-23ff-11e9-9f66-e8cd68dcacc5.png)

Reply:
Good catch!